tests/test_forms.py

```diff
@@ -236,10 +236,27 @@ def test_setting_of_field_properties(self):
             self.assertEqual(title_de_field.widget.__class__, forms.widgets.Textarea)
 
             title_fr_field = form.fields["title_fr"]
-            self.assertEqual(title_fr_field.label, "Title (FR, default language)")
+            self.assertEqual(title_fr_field.label, "Title (FR, fallback language)")
             self.assertEqual(title_fr_field.required, True)
             self.assertEqual(title_fr_field.widget.__class__, forms.widgets.Textarea)
 
+        with self.subTest("Custom labels"):
+
+            class CustomLabelForm(TranslationModelForm):
+                class Meta:
+                    model = Challenge
+                    fields = ["title"]
+                    languages = ["browser", "fallback"]
+                    fallback_language = "fr"
+                    fallback_label = "default language"
+                    translation_label = "new language"
```

Comment:
Or do we prefer something more compact like:
```suggestion
                    labels = {"fallback_label": "default language", "translation_label": "new language"}
```